mypy/stubtest.py

```diff
@@ -540,6 +543,186 @@ def names_approx_match(a: str, b: str) -> bool:
     )
 
 
+class _NodeEvaluator(ExpressionVisitor[object]):
```

Comment:
```suggestion
class _NodeEvaluator(object, ExpressionVisitor[object]):
```

Reply:
Tried that, it doesn't even work without mypyc because it creates an inconsistent MRO. I'll try the generic way you suggested instead.